apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/MatrixQuestionForm.tsx

```diff
@@ -0,0 +1,180 @@
+"use client";
+
+import { MatrixLabelInput } from "@/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/MatrixInput";
+import { PlusIcon, TrashIcon } from "lucide-react";
+import { useState } from "react";
+import { toast } from "react-hot-toast";
+
+import { TSurvey, TSurveyMatrixQuestion } from "@formbricks/types/surveys";
+import { Button } from "@formbricks/ui/Button";
+import { Label } from "@formbricks/ui/Label";
+import QuestionFormInput from "@formbricks/ui/QuestionFormInput";
+
+interface MatrixQuestionFormProps {
+  localSurvey: TSurvey;
+  question: TSurveyMatrixQuestion;
+  questionIdx: number;
+  updateQuestion: (questionIdx: number, updatedAttributes: any) => void;
```

Comment:
Can we typecast the `updatedAttributes` away from any please?

Reply:
Yes this was the case with every other questions as well, changed them too 😊